backend-shared/config/db/utils.go

```diff
@@ -487,15 +491,36 @@ func SetupForTestingDBGinkgo() error {
 
 			Expect(err).To(BeNil())
 			if err == nil {
-				Expect(rowsAffected).Should((Equal(1)))
+				Expect(rowsAffected).Should(Equal(1))
 			}
 		}
 	}
 
 	return nil
 }
 
-// Non-Ginkgo-based tests that need to setup the database should call this first.
+func removeAnyRepositoryCredentialsTestEntries(dbq AllDatabaseQueries, ctx context.Context) error {
```

Comment:
One more thing, the Go idiom (AFAIK) is that `ctx context.Context` should always be the first parameter.

Reply:
you are correct, thanks for spotting it ;)